roles/nat64_appliance/tasks/main.yml

```diff
@@ -66,12 +66,31 @@
     dest: "{{ cifmw_nat64_appliance_workdir }}/edpm-image-builder"
     version: main
 
+- name: Find latest image name
+  when:
+    - cifmw_discovered_image_url is not defined
+    - not cifmw_edpm_build_images_dry_run
+  ansible.builtin.import_role:
+    name: discover_latest_image
```

Comment:
Does'nt this mean we would be using a RHEL image vs CentOS image - depending on what parameters are set up for `discover_latest_image`? Are we sure a RHEL image will work? 

I think for a RHEL image to to work we would have to update https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/nat64_appliance/files/nat64-appliance.yaml 

What issue are we trying to fix? 
There are bugs, and fixes proposed for diskimage-builder related to CentOS image changes - I am guessing, but if the reason for this change is due to those bugs we should instead focus on fixing that, and not change this role.

-  https://bugs.launchpad.net/ironic/+bug/2120974
- https://review.opendev.org/c/openstack/diskimage-builder/+/957983